lib/utilities/is-yarn-project.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const path = require('path');
+const existsSync = require('exists-sync');
+const findWorkspaceRoot = require('find-yarn-workspace-root');
+
+/**
+ * Returns true iff the given directory has a yarn.lock file or is a child of a
```

Comment:
`iff` => `if`

Reply:
Iff was actually intentional (not that it really matters) — https://en.m.wikipedia.org/wiki/If_and_only_if 😁

Reply:
if I met this "only if" I'd make a pr to fix a typo